src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -150,6 +159,18 @@ private void addDxAddressToContactInformation(List<DxAddressCreationRequest> dxA
         }
     }
 
+    private void persistedUserAccountMap(ProfessionalUser persistedSuperUser, List<PaymentAccount> paymentAccounts) {
+
+        if (paymentAccounts != null
```

Comment:
CollectionUtils.isEmpty method can be used

Reply:
in the next JIRA i will implement the above change